lib/zold/commands/node.rb

```diff
@@ -287,7 +287,9 @@ def nohup(opts)
           begin
             code = exec("#{myself} #{args.join(' ')}", nohup_log)
             raise "Exit code is #{code}" if code != 0
-            exec(opts['nohup-command'], nohup_log)
+            Open3.popen3('gem uninstall zold -a --ignore-dependencies') do
+              exec(opts['nohup-command'], nohup_log)
```

Comment:
@emilianodellacasa this looks a bit weird... First, we won't see the output of that `uninstall`. Second, what if the user wants to change the command and do `sudo gem uninstall`? It's impossible, right? I would better simply  change the default command in `nohup-command` to something like `gem uninstall && gem install`

Reply:
@yegor256 What if we add another option to ./zold to change the uninstall command to whatever the user wants? Also, I don't see the problem in not showing the output of the uninstall, the user won't see it anyway. Also, by putting the uninstall command in the nohup-command, user could skip that part and I think it is important that there is only one instance of Zold gem running.